### ISSUE TITLE
Correct version of rust is not accepted when using rustup

### ISSUE THREAD

**Issue by NobbZ** [hansihe/Rustler#15]
This is the ouput of a fresh mix project, where just the things from the documentation have been added.

```
$ mix compile
warning: 'multirust' is being renamed to 'rustup'. this command is deprecated. use 'rustup' instead
Required Rust toolchain version nightly-2016-05-07 is not installed.
It can be installed by running 'multirust update nightly-2016-05-07'.

** (RuntimeError) Compilation error
    lib/compiler.ex:9: Mix.Tasks.Compile.Rustler.throw_error/1
    lib/compiler.ex:22: Mix.Tasks.Compile.Rustler.run/1
    (elixir) lib/enum.ex:1105: Enum."-map/2-lists^map/1-0-"/2
    (mix) lib/mix/tasks/compile.all.ex:19: anonymous fn/1 in Mix.Tasks.Compile.All.run/1
    (mix) lib/mix/tasks/compile.all.ex:37: Mix.Tasks.Compile.All.with_logger_app/1
    (mix) lib/mix/tasks/compile.ex:84: Mix.Tasks.Compile.run/1
    (mix) lib/mix/cli.ex:58: Mix.CLI.run_task/2

$ rustup show
installed toolchains
--------------------

nightly-2016-05-07-x86_64-unknown-linux-gnu
nightly-x86_64-unknown-linux-gnu (default)

active toolchain
----------------

nightly-2016-05-07-x86_64-unknown-linux-gnu (directory override for '/home/NobbZ/projects/elixir/rustler_foo')
rustc 1.10.0-nightly (62e2b2fb7 2016-05-06)


$ rustc --version
rustc 1.10.0-nightly (62e2b2fb7 2016-05-06)
```

**Comment by hansihe** [comment]
That is because only multirust is supported at the moment. Because of the recent mutlirust deprecation, adding rustup support is definitely planned.

**Comment by hansihe** [comment]
Fixed along with #14
